tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -123,6 +123,8 @@ At the end of the `AsyncMain` method, after the configuration code, add the foll
 
 snippet:Startup
 
+NOTE: In this course we will always use `.ConfigureAwait(false)` when awaiting tasks, in order to [avoid capturing and restoring the SynchronizationContext](/nservicebus/handlers/async-handlers.md#usage-of-configureawait).
```

Comment:
@SimonCropp this is consistent with the rest of docs right? This is the first place where ConfigureAwait appears in the lessons.